### ISSUE TITLE
Result does not sort correctly with Multiple OrderBy

### ISSUE THREAD

**Issue by divya-sea** [grafana/grafana-iot-twinmaker-app#128]
1. Steps:
        1. Select a component, and enter property group / selected properties / filter. 
        2. Add multiple order by and run query
    2. Error:
        1. Response on the plugin is not sorted based on the order of the OrderBy.
        2. Example add first order as x and second order as Y.
        3. Display on the panel order is based on Y but not on X.
        4. We need to add a logic sort based on the order of the OrderBy.

**Comment by toddtreece** [comment]
closing this one. as far as i can tell this seems to be an API bug. here's the request the plugin is sending:

```json
{
    "componentName": "TabularComponent",
    "entityId": "<redacted>",
    "maxResults": 200,
    "propertyGroupName": "tabularPropertyGroup",
    "selectedProperties": [
        "total",
        "crit"
    ],
    "tabularConditions": {
        "orderBy": [
            {
                "order": "ASCENDING",
                "propertyName": "total"
            },
            {
                "order": "ASCENDING",
                "propertyName": "total"
            }
        ]
    }
}
```

here's a truncated response:

```json
{
    "propertyValues": {},
    "tabularPropertyValues": [
        [
            {
                "crit": {
                    "integerValue": 4
                },
                "total": {
                    "integerValue": 0
                }
            },
            {
                "crit": {
                    "integerValue": 5
                },
                "total": {
                    "integerValue": 0
                }
            },
            {
                "crit": {
                    "integerValue": 4
                },
                "total": {
                    "integerValue": 0
                }
            }
        ]
    ]
}
```

**Comment by toddtreece** [comment]
scratch that, i think i found the issue. it seems to be a state issue on the frontend that is causing duplicate property names to be sent in the request:

```json
           {
                "order": "ASCENDING",
                "propertyName": "total"
            },
            {
                "order": "ASCENDING",
                "propertyName": "total"
            }
```

**Comment by divya-sea** [comment]
[AP response.pdf](https://github.com/grafana/grafana-iot-twinmaker-app/files/10009112/AP.response.pdf)

**Comment by toddtreece** [comment]
fixed by #129
